src/components/menu-bottom/MenuBottom.vue

```diff
@@ -39,14 +40,19 @@ export default Vue.extend({
   data: (): {
     toolDataList: ToolData[];
     toolSelectedIndex: number;
-    temporaryTool: BaseTool | null;
   } => ({
     toolDataList: [
       {
-        label: "Pan and Zoom (Hold Ctrl/Meta to turn on)",
+        label: "Select Box and Move",
```

Comment:
Can you add a note to hold ctrl/meta to pan, shift to add to the selection, and shift + option/alt to toggle selection?

Reply:
This gets really wordy.  Now all the tools have pan, does it need to be called out?

I'm going to leave out the pan/scoll to zoom as they all work this way, but instead call out what the tools individually do.

Reply:
It made things very wordy to add this.  I've removed the note about pan and zoom as they apply to all the moves, but instead highlighted how they were different.